bridgesample/src/main/AndroidManifest.xml

```diff
@@ -28,14 +28,50 @@
 
         <activity android:name=".scenario.activity.FragmentContainerActivity" />
 
-        <activity android:name=".scenario.activity.LargeDataActivity" />
+        <activity android:name=".scenario.activity.LargeDataActivity"
+            android:launchMode="singleTask">
```

Comment:
So I see why you'd like to make this `singleTask` but I think we should instead make a new Activity specifically for this purpose. `DeepLinkLargeDataActivity` or something like that.